markdown/src/Makefile.am

```diff
@@ -35,4 +35,7 @@ markdown_la_CFLAGS += $(LIBMARKDOWN_CFLAGS)
 markdown_la_LIBADD += $(LIBMARKDOWN_LIBS)
 endif
 
+if WEBKIT2
+markdown_la_CFLAGS += -DWEBKIT2
```

Comment:
If above is a typo, maybe it would be good to add the `MARKDOWN_` prefix for this CPP define too, in case of any kind of conflict with a macro in the Webkit source? I don't care either way, but macros can get nasty when they clash.